spec/features/answers/answers_have_comments_spec.rb

```diff
@@ -1,10 +1,16 @@
 require 'rails_helper'
 
 feature 'user views a question' do
+  attr_reader :user, :question, :answer
+  before(:each) do
+    @user = Fabricate(:user, password: 'password')
+    @user.roles.create(name: 'registered_user')
```

Comment:
I did my best in trying to work with Fabricator to make that happen, but I couldn't find a way to pull it off. If we can find some time, I'm happy to work together with you on it.